documentation/anonymization.markdown

```diff
@@ -9,11 +9,28 @@ The export tool has the ability to anonymize the PuppetDB archive before returni
 Using the `export` command
 -----
 
-To create an anonymized PuppetDB archive directly, simply run the `puppetdb export` tool, preferably on your PuppetDB instance:
+To create an anonymized PuppetDB archive directly, use the Puppet `db` subcommand
+from any node with puppetlabs-client-tools installed:
```

Comment:
For PE it's pe-client-tools, @jdwelch for the FOSS version are we calling this puppet-client-tools?